### ISSUE TITLE
Teleport? 

### ISSUE THREAD

**Issue by MThogersen** [DrBeef/Lambda1VR#81]
Great port! Is it possible to add a teleport-like locomotion? Thanks for all the work put into this!

**Comment by neitey** [comment]
Dr beef said he won't do it, but others are free to fork.

MThogersen <notifications@github.com> schrieb am So., 19. Apr. 2020, 14:07:

**Comment by JoLlyl0l** [comment]
Yeah, usual locomotion makes sick pretty fast

**Comment by Alex20129** [comment]
Yeah. For me slide/strafe causes sickness in just 10~15 minutes. I'm not sure if teleport can completely solve this problem, but it's good to have it as an option.
